### ISSUE TITLE
Prepublish checklist error message on Video Alt text missing is incorrect.

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#5405]
## Bug Description

As Videos don't have Alt text (see #5332 ) we should not be presenting prepublish checklist messages about missing Alt text
![Screen Shot 2020-11-23 at 2 47 10 PM](https://user-images.githubusercontent.com/58738530/100007963-d19ef080-2d9a-11eb-8fda-40d25db7aa7c.png)
source - https://stories-new-wordpress-amp.pantheonsite.io/wp-admin/post.php?action=edit&post=13619#page=%2522b1abc9e1-6802-4027-a3eb-7782cf5ceca5%2522

Instead this message should be updated to display the following when no video description has been provided:

Video missing description 
Add a video description. Descriptions help with indexability and accessibility (for videos without captions). Include any burned-in text inside of the video.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Tested 5417

Message re: missing description not appearing (and seeing "too small" message appear twice):
![image.png](https://images.zenhubusercontent.com/5ea085e4704e72c0a99c6997/12d2ccea-af2d-4c8c-b717-15651b4aa535)

**Comment by carloskelly13** [comment]
![Screen Shot 2020-12-07 at 1.43.51 PM.png](https://images.zenhubusercontent.com/5ea09bca6a44046f869bafe0/4dab62d3-3019-4d39-a9be-009379176323)

This was fixed in the image PR; so this is somewhat a duplicate ticket as well. If we do not see this working on main, we can re-open this. The duplicate message is another bug ticket that is being addressed.

**Comment by csossi** [comment]
Checking Main and seeing this:
![image.png](https://images.zenhubusercontent.com/5ea085e4704e72c0a99c6997/3511f111-80f0-42bc-a68c-13c245358fd9)

Which doesn't match the copy above

```
Video missing description
Add a video description. Descriptions help with indexability and accessibility (for videos without captions). Include any burned-in text inside of the video.
```

**Comment by swissspidy** [comment]
This is somewhat confusing. There are currently two separate checks here, one for missing `alt` and one for missing `title`, with two different messages.

https://github.com/google/web-stories-wp/blob/d7223976202e7cb2a89c623d9d0a04e99abaab20/assets/src/edit-story/app/prepublish/constants.js#L163-L169

https://github.com/google/web-stories-wp/blob/d7223976202e7cb2a89c623d9d0a04e99abaab20/assets/src/edit-story/app/prepublish/constants.js#L191-L198

Not sure why only one was updated, but we're actually combining these two fields into one in #5332, so we can keep the one with the right messaging.

Since this is already in `main`, I suggest to leave this to #5332 to address.

**Comment by csossi** [comment]
Verified in QA
